docker/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
+
+RUN yum install -y php php-cli php-pdo php-mbstring php-intl php-pdo \
+	php-pgsql php-mysqlnd php-xml git && \
+	curl -sS https://getcomposer.org/installer | php && \
+	mv composer.phar /usr/local/bin/composer
+
+COPY docker/entrypoint.bash /usr/local/bin/entrypoint.bash
+COPY composer.json /vendor_preinstalled/
+
+RUN cd /vendor_preinstalled/ && \
+	composer update --no-interaction --prefer-source --prefer-stable \
+		--prefer-lowest
+
+WORKDIR /app/
+ENTRYPOINT ["/bin/bash", "/usr/local/bin/entrypoint.bash"];
```

Comment:
Is really necessary use that complex entrypoint? From my point of view (after different project migrated to Docker development) is better switch devs directly to the command line when common commands are present.

Something like `docker-compose run phinx bash` and there is prepared tools like `composer` and others.
On the other side you can use [composer scripts](https://getcomposer.org/doc/articles/scripts.md) for that commands.